org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -1,4 +1,6 @@
 @import '../../../org.civicrm.bootstrap/scss/vendor/bootstrap/buttons';
+@import '../../../org.civicrm.bootstrap/scss/vendor/bootstrap/button-groups';
```

Comment:
This is already done in #1337, but i needed to do it here as well because I don't which PR is going to be merged first